### PR TITLE
Point Tests to RSKj master (Iris) instead of PAPYRUS

### PR DESCRIPTION
**Pull request by jjtechuy**
- Added parameters for repo, branch and release at config.yml
- Pointed tests to go against RSKj/master instead of PAPYRUS

### CODE REVIEW

**Review by diegomasini — CHANGES_REQUESTED** [review]
CI tests are failing with latest change to use master instead of PAPYRUS-2.0.1. I cannot approve this PR until tests are passing again.

File: .circleci/config.yml
Comment (mortelli) [inline .circleci/config.yml:222]: may i ask why this was added only to test thread #6? 

(not entirely sure what the purpose of this is, but mainly curious)

File: test/relayserver/RelayServer.test.ts
Comment (mortelli) [inline test/relayserver/RelayServer.test.ts:30]: rather than having this `const` as `true`, wouldn't it be better to remove it and have the affected test like this?:

```typescript
    describe('#validateViewCallSucceeds()', function () {
      // RelayHub contract
      it('should fail to relay rejected transaction', async function () {
        const req = await env.createRelayHttpRequest()

        req.metadata.signature = INCORRECT_ECDSA_SIGNATURE
        const method = env.relayHub.contract.methods.relayCall(req.relayRequest, req.metadata.signature)

        try {
          await env.relayServer.validateViewCallSucceeds(method, req, 2000000)
          assert.fail()
        } catch (e) {
          assert.include(e.message, 'signature mismatch')
        }
      })
    })
```

**Review by mortelli — APPROVED** [review]
checks are passing, so in principle it looks good :+1: 

leaving a couple of comments down below and some more here.

firstly, @JONAF2103 and @mauricioirace have implemented the following auxiliary function:

https://github.com/rsksmart/enveloping/blob/f7a4e28f9d8a214f15a30753d4a2e7e42fbb9b0a/test/TestUtils.ts#L497-L502

this could be an alternative to the changes made in this PR to files `test/EnvelopingUtils.test.ts`, `test/relayclient/RelayClient.test.ts` and `test/relayclient/RelayProvider.test.ts`.

lastly: do you think would it be prudent to wait for the official Iris release before merging this PR?

thanks

**Review by raullaprida — APPROVED** [review]
LGTM!

**Review by diegomasini — APPROVED** [review]
LGTM!

**Pull request merged**
